cmd/cloud-controller-manager/app/options/options.go

```diff
@@ -86,6 +86,7 @@ func (s *CloudControllerManagerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&s.EnableProfiling, "profiling", true, "Enable profiling via web interface host:port/debug/pprof/.")
 	fs.BoolVar(&s.EnableContentionProfiling, "contention-profiling", false, "Enable lock contention profiling, if profiling is enabled.")
 	fs.StringVar(&s.ClusterCIDR, "cluster-cidr", s.ClusterCIDR, "CIDR Range for Pods in cluster.")
+	fs.StringVar(&s.ClusterName, "cluster-name", s.ClusterName, "The instance prefix for the cluster.")
```

Comment:
This doesn't explain what the flag is for.

Reply:
@thockin, it's just a cut-n-paste from the kube-controller-manager

https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-controller-manager/app/options/options.go#L193

Reply:
I agree with both @thockin and @dims. This should be fixed, but as it is a copy-paste from kcm it is ok for this v1.8 bugfix. Can be more descriptive in future releases.